### ISSUE TITLE
Is this package allowed by Admob?

### ISSUE THREAD

**Issue by nathanael540** [duyduong/flutter_native_admob#34]
I started using this package in my application and since then I have been facing some strange things in my report.

My data simply lost its reason. My clicks basically zeroed out during the afternoon, even with a large number during the previous day.

My eCPM varies and I no longer see any pattern of growth or decline. Each day is literally a new day. In heights, or in darkness.

I wonder if Admob was marking clicks and views as invalid traffic. And I fear for the banning of my account.

**Comment by cristighr1995** [comment]
Do you still have this issue? Is it safe to use this package for native ads?

**Comment by nathanael540** [comment]
Yes. My app was only targeting CPM Ads. I dont know how, but after I used this pack my app is only showing the CPM ads.

I work with admob in 3 years ago, and never see this.

I unlist my app on Play Store and I’m developing one new version in another platform that is not Flutter to try get my clicks back.

Sorry for my English. Im Brazilian

**Comment by duyduong** [comment]
Hey guys, I don't know what do u mean by allow. I used this in my app and it works as expected
Notice that this is just native ads
